### PR TITLE
01-question added

### PR DESCRIPTION
**Pull request by gitvibav**
Please verify whether the format is correct.

### CODE REVIEW

File: 01-valid-number.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## 65. Valid Number
```
Comment (ljeng) [inline 01-valid-number.md:1]: Remove 65.

File: 01-valid-number.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Formally, a **valid number** is defined using one of the following definitions:
+
+1. An **integer number** followed by an optional exponent.
+2. A **decimal number** followed by an optional exponent.
+
+An **integer number** is defined with an optional sign (`-` or `+`) followed by digits.
+
+A **decimal number** is defined with an optional sign (`-` or `+`) followed by one of the following definitions:
+1. Digits followed by a dot (`.`).
```
Comment (ljeng) [inline 01-valid-number.md:15]: Insert space before list

File: 01-valid-number.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+2. Digits followed by a dot (`.`) followed by digits.
+3. A dot (`.`) followed by digits.
+
+An exponent is defined with an exponent notation (`e` or `E`) followed by an integer number.
+
+The digits are defined as one or more digits.
```
Comment (ljeng) [inline 01-valid-number.md:21]: Insert blank line at the end of file

**Review by ljeng — CHANGES_REQUESTED** [review]

**Review by gitvibav — COMMENTED** [review]
I've made the necessary changes

**Pull request merged**